website/django/data/migrations/0035_merge.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
I agree that the apparently duplicated migration is confusing -- it's because this branch was originally based on master before Marc added that migration. The "empty" migration is a merge migration, created via `./manage.py makemigrations --merge`, which reconciles the two colliding migrations. It passed the tests because it's technically correct, but in retrospect pretty confusing.

Anyway, I think the better thing to do would be to remove both migrations that I added, rebase from master so that I'm sure all the migrations are in, and then regenerate it as a single, new migration so that we have a linear history.

Reply:
Ok, I've reverted the migration history to 0032, applied 0033 (Marc's migration), and re-added a single migration for insilicopriors (0034) that keeps the migration history linear.

Reply:
gotcha, I figured this was probably a django thing but I think your adjustments are much more dev friendly, thanks!